recipe-server/client/control/selectors/FiltersSelector.js

```diff
@@ -132,7 +132,7 @@ export const isFilteringActive = state =>
  * Get all filters as an object keyed on their slug
  *
  * @param  {Array<Object>} groups All filter groups
```

Comment:
D'oh, sorry.

The reason for using `list` was that it was more descriptive of what the function expected (it wants `state.filters.list`). However, looking at the other selectors in that file, they all expect the general `state` object, and then pull out what they need from there. I've changed `getFilterObject` to now accept the `state` object to conform to conventions.

I also noticed that a bunch of the docstrings were inaccurate, so I corrected those while I was in there, as well.